File: app/src/main/java/com/example/rickmortymvvm/intrefaces/CharacterService.java
package com.example.rickmortymvvm.intrefaces;

import com.example.rickmortymvvm.CharacterResponseVO;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

    public interface CharacterService {
        String BASE_URL = "https://rickandmortyapi.com/api/";
        @GET("character/")
        Call<CharacterResponseVO> listCharacter(@Query("page") int currentPage );
    }

File: app/src/main/java/com/example/rickmortymvvm/services/CharacterService.kt
package com.example.rickmortymvvm.services

import com.example.rickmortymvvm.services.models.CharacterResponseVO
import io.reactivex.Observable

import retrofit2.http.GET
import retrofit2.http.Query

interface CharacterService {
    @GET("character/")
    fun listCharacter(@Query("page") currentPage: Int): Observable<CharacterResponseVO?>
}
